neurons/miners/StableMiner/base.py

```diff
@@ -109,8 +109,11 @@ def start_axon(self):
             bt.axon(
                 wallet=self.wallet,
                 ip=bt.utils.networking.get_external_ip(),
-                external_ip=bt.utils.networking.get_external_ip(),
-                config=self.config
+                external_ip=self.config.axon.get(
```

Comment:
Should this be:
`external_ip=self.config.axon.get("external_ip") or bt.utils.networking.get_external_ip(),`

?